packages/windows/src/RPA/Windows/keywords/locators.py

```diff
@@ -241,6 +246,17 @@ def _get_element_with_locator_part(
         new_element.robocorp_click_offset = offset
         return new_element
 
+    def _load_by_alias(self, criteria: str) -> str:
+        try:
+            locator = LocatorsDatabase.load_by_name(criteria, self._locators_path)
+            if isinstance(locator, WindowsLocator):
+                return locator.value
+        except ValueError:
+            # How to check if locator check should be done as inspector
```

Comment:
This could be handled more nicely with in the future if we harmonize the syntax parsing between `RPA.WIndows` and `RPA.Desktop`